component/scripts/copy-pg-backup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+xrdname=$(kubectl -n "${CLAIM_NAMESPACE}" get vshnpostgresqls "${CLAIM_NAME}" -ojson | jq -r '.spec.resourceRef.name')
+
+source_namespace=$(kubectl -n "${CLAIM_NAMESPACE}" get vshnpostgresqls "${CLAIM_NAME}" -ojson | jq -r '.status.instanceNamespace')
+
+echo "copy secret"
+kubectl -n "${source_namespace}" get secret "pgbucket-${xrdname}" -ojson | jq 'del(.metadata.namespace)' | kubectl -n "${TARGET_NAMESPACE}" apply -f -
```

Comment:
Where the `TARGET_NAMESPACE` comes from?

Reply:
It gets patched in the composition.